### PR TITLE
Feature/feedback form

### PR DESCRIPTION
**Pull request by roxanap33**

### CODE REVIEW

File: frontend/src/pages/feedback/FeedbackForm.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+export const InputContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  padding: ${rem(20)} ${rem(0)};
+`;
+
+export const StyledLabel = styled.label`
+  font-size: ${rem(18)};
+  margin-bottom: 5px;
```
Comment (adrianacostep33) [inline frontend/src/pages/feedback/FeedbackForm.styled.tsx]: Add the margin in rem

File: frontend/src/pages/feedback/FeedbackForm.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+`;
+
+export const StyledInput = styled.input`
+  background: #575c7b;
+  border-radius: ${rem(8)};
+  height: ${rem(25)};
+  width: ${rem(350)};
+  color: ${pallete.PLATINUM};
+  padding: 20px;
```
Comment (adrianacostep33) [inline frontend/src/pages/feedback/FeedbackForm.styled.tsx]: Please add rem here

File: frontend/src/pages/feedback/FeedbackForm.styled.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+  @media (max-width: 550px) {
+    width: ${rem(300)};
+  }
+`;
+
+export const FeedbacFormRatingSection = styled.div`
+  display: flex;
+  gap: 15px;
```
Comment (adrianacostep33) [inline frontend/src/pages/feedback/FeedbackForm.styled.tsx]: Also here

File: frontend/src/pages/feedback/FeedbackForm.styled.tsx [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+  padding: ${rem(20)} ${rem(0)};
+`;
+
+export const StyledLabel = styled.label`
+  font-size: ${rem(18)};
+  margin-bottom: 5px;
+`;
+
+export const StyledInput = styled.input`
+  background: #575c7b;
```
Comment (ioanagirbap33) [inline frontend/src/pages/feedback/FeedbackForm.styled.tsx]: You didn't use the variable for the color

File: frontend/src/pages/feedback/FeedbackForm.tsx [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+                    />
+                  </label>
+                );
+              })}
+            </FeedbacFormRatingSection>
+            <Input
+              label="Date"
+              defaultValue={getReviewDate(formattedDateTime)}
+              disable={true}
+            ></Input>
```
Comment (adrianacostep33) [inline frontend/src/pages/feedback/FeedbackForm.tsx]: Do you need the closing tag?

**Review by ioanagirbap33 — APPROVED** [review]

**Review by adrianacostep33 — APPROVED** [review]

**Pull request merged**